internal/endpoints/endpointsFetcher.go

```diff
@@ -0,0 +1,60 @@
+package endpoints
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strings"
+)
+
+// Schema for data pulled out of EndpointSources file
+type ListOfEndpoints struct {
+	Entries []struct {
+		OrganizationName     string `json:"OrganizationName"`
```

Comment:
I'm curious about the `json:"OrganizationName"` syntax. Please explain to me. If this is typical go, it doesn't need a comment.

Reply:
That is a struct-tag https://github.com/golang/go/wiki/Well-known-struct-tags it tells the parsers more information about what/where to find information for that field.

Reply:
BTW this struct was generated using https://mholt.github.io/json-to-go/